CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
 
+## Unreleased as of Sprint 96 ending 2018-10-08
+
+### Added
+- Move from apache module mod_auth_kerb to mod_auth_gssapi [(#206)](https://github.com/ManageIQ/manageiq-appliance/pull/206)
+
+### Fixed
+- Add max-time to curl commands [(#209)](https://github.com/ManageIQ/manageiq-appliance/pull/209)
```

Comment:
How is it that this change was added to both the "Unreleased" section here and also the "Released" section [here](https://github.com/ManageIQ/manageiq-appliance/pull/211/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR21). A change can't be both released and unreleased. In fact, why aren't the hammer beta releases in this file at all?

Reply:
This really applies to all of the master/hammer change logs it looks like.

I'll hold off on merging anything until this gets resolved here.

@Fryguy thoughts?

Reply:
@carbonin I haven't updated master branch just yet to bring over the Hammer Beta 1 updates. (this is what has been done in the past) Chicken and egg thing. The hammer branch had to take precedence. My plan was to update master either this week, or better, wait for next week's sprint and do it all in one shot.

Reply:
This sounds exactly backwards of how I would expect this to be done. Typically changes are made to master first then backported to the release branches. Why would this be any different?

Reply:
Couldn't we make a PR to master titled "Add Hammer Beta 1.1 to the Changelog" then label it hammer/yes and have @simaishi backport it like she does for everything else?